src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Locale;
+import java.util.Random;
+
 public class ColorSupplier {
+    public static final int lengthOfEnum = Color.values().length;
+    private int index = new Random().nextInt(lengthOfEnum);
```

Comment:
You can use Color.values().length directly where needed. 'index' shouldn't be a class field as it's supposed to be random. With your current solution it isn't. Make Random a class field